src/Helper/EntriesExport.php

```diff
@@ -300,7 +300,7 @@ private function producePdf(): Response
         $pdf->AddPage();
         $html = $this->getExportInformation('tcpdf');
 
-        $pdf->writeHTMLCell(0, 0, '', '', $html, 0, 1, 0, true, '', true);
```

Comment:
should have changes the last 0 by false, but then last 4 parameters are default values, so removed